### ISSUE TITLE
About the pretrained model

### ISSUE THREAD

**Issue by Terminal-K** [haomo-ai/MotionSeg3D#11]
Hi, i used the pretrained weights which you provided to predict, and i get the result as follow:
![z1Ldg8cozx](https://user-images.githubusercontent.com/59523614/206655375-0044ef1c-0631-4483-91fe-ac3f98851707.jpg)
It doesn't seem to be the weight in the paper

**Comment by MaxChanger** [comment]
Thanks for your interest in our project.
Could you provided more details about the command you used? 
And which pretrianed model used for inference ?

**Comment by Terminal-K** [comment]
The pretrained weights i used is from here
![R3IRCi1nw1](https://user-images.githubusercontent.com/59523614/206662893-523ec40d-ab68-485b-9c68-2a35b08e1115.jpg)

i first inferred on dataset by running

> python3 ./infer.py -d Dataset -m ./pretrainedModel -l ./valid -s valid

and then i used this command to evaluate 
> python3 ./utils/evaluate_mos.py -d Dataset -p ./valid --datacfg ./config/labels/semantic-kitti-mos.yaml

**Comment by MaxChanger** [comment]
Ok, I'm going to retest, but since my gpu server crashed the other day, I need to reconfigure the environment. please hold on

**Comment by MaxChanger** [comment]
Hi, sorry for the long wait, I found out that the problem is because `nan` appears during inference. 
This problem is also mentioned in https://github.com/haomo-ai/MotionSeg3D/issues/9 and https://github.com/haomo-ai/MotionSeg3D/issues/6, I will debug why this happens

**Comment by MaxChanger** [comment]
Hi @Terminal-K , I finally found the cause of this problem!
I ignored the version of softpool, its author updated the repo on 2022/04/07, and our project was implemented before that, using its historical version, the commits id is `2d2ec6d`.

When I use the new version `d056ab8` of softpool code for inference, nan will also appear, just like what you encounter in the process of inference or train.

The specific reason may need to carefully check the softpool code and discuss with the author, but rolling back the softpool version is a quick solution to this problem.


```shell
git clone https://github.com/alexandrosstergiou/SoftPool.git
cd SoftPool
git checkout 2d2ec6d # rollback to 2d2ec6dca10b7683ffd41061a27910d67816bfa5
cd pytorch
make install
--- (optional) ---
make test
```
I hope you can help to check it, if you have any questions, please contact me again.

**Comment by Terminal-K** [comment]
Thank you for your patience, i will try again as you say:)
